### PR TITLE
Feat/live 2804/asset header graph v0

### PR DESCRIPTION
**Pull request by mlegall-ledger**
### 📝 Description

Created GraphCard, header and the general upper layout of the new Wallet Page.
this is an intermediate merge, there are still some elements to add and pixel polish to be done

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

Screenshot 2022-08-11 at 16 02 58

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/AccountGraphCard.tsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ function AccountGraphCard({
           <InfiniteLoader size={32} />
         )}
       </Flex>
-      <Flex pt={16} px={6} bg={colors.background.main}>
+      <Flex bg={colors.background.main}>
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/AccountGraphCard.tsx:175]: Tu peux faire directement bg="background.main"n sans devoir récuperer l'object colors via useTheme du coup
la string est convertie en propriété par styled-system

File: apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+                    <>
+                      <SmallPlaceholder mt="12px" />
+                    </>
+                  ) : (
+                    <Flex flexDirection="row" alignItems="center">
+                      {hoveredItem && hoveredItem.date ? (
+                        <Text
+                          variant={"body"}
+                          fontWeight={"semibold"}
+                          fontSize="16px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx:186]: variant=large and remove fontSize 16px

File: apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+                        joinFragmentsSeparator=" "
+                      />
+                    </Text>
+                  )}
+                  <TransactionsPendingConfirmationWarning />
+                </Flex>
+                <Flex flexDirection={"row"}>
+                  {!balanceAvailable ? (
+                    <>
+                      <SmallPlaceholder mt="12px" />
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx:178]: use theme variable
mt="12px" => mt=4

File: apps/ledger-live-mobile/src/components/AccountGraphCard.tsx [hunk 4]
```diff
@@ -223,12 +223,7 @@ const GraphCardHeader = ({
   }
 
   return (
-    <Flex
-      flexDirection={"row"}
-      px={6}
-      pt={100}
-      justifyContent={"space-between"}
-    >
+    <Flex flexDirection={"row"} pt={100} justifyContent={"space-between"}>
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/AccountGraphCard.tsx:226]: pt=13

File: apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx [hunk 5]
```diff
@@ -42,7 +42,7 @@ export default function AccountsNavigator() {
       />
       <Stack.Screen
         name={ScreenName.Account}
-        component={readOnlyModeEnabled ? ReadOnlyAccount : Account}
+        component={readOnlyModeEnabled ? ReadOnlyAccount : WalletCentricAsset}
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx:45]: You can put back account here

**Pull request merged**